### DISCUSSION TITLE
At now we can use structuredClone instead _deepClone

### DISCUSSION THREAD

**Discussion by budarin** [pmndrs/valtio#854] [Ideas]
structuredClone is supported by all modern browsers

https://caniuse.com/?search=structuredClone

**Comment by dai-shi** [comment]
Feel free to use it where you can. Do our docs mention about `_.deepClone`?

**Comment by budarin** [comment]
Yes, I wrote this because I met _deepClone in the documentation

**Reply by dai-shi** [reply]
I see it's `_.cloneDeep`. Feel free to open a PR to improve docs.
